### PR TITLE
fixes exception raised from ansible-connection logging

### PR DESCRIPTION
**Pull request by privateip**
updates ansible-connection logging functions to use display instead of
log.

### CODE REVIEW

File: bin/ansible-connection [hunk 1]
```diff
@@ -290,18 +290,18 @@ def main():
             try:
                 server = Server(sf_path, pc)
             except AnsibleConnectionFailure as exc:
-                log(str(exc), pc.remote_addr, pc.remote_user)
+                display.dislay(str(exc), pc.remote_addr, pc.remote_user, log_only=True)
```
Comment (rcarrillocruz) [inline bin/ansible-connection]: this should be display.display?

File: bin/ansible-connection [hunk 2]
```diff
@@ -290,18 +290,18 @@ def main():
...
                 rc = 1
             except Exception as exc:
-                log(traceback.format_exc(), pc.remote_addr, pc.remote_user)
+                display.dislay(traceback.format_exc(), pc.remote_addr, pc.remote_user, log_only=True)
```
Comment (rcarrillocruz) [inline bin/ansible-connection]: this should be display.display?

**Review by rcarrillocruz — COMMENTED** [review]
I believe you have some typos

File: bin/ansible-connection (hunk 1 above)
Comment (privateip) [reply bin/ansible-connection]: good catch

**Pull request merged**